### PR TITLE
[PL] Breach Scanner & Mimicry, LoadingHints, Normalizer and other minor changes

### PR DESCRIPTION
**Pull request by TTypiarz**
# **SCP:SL Translation PR Template**

## **Language Name**
Polish

## **State Type of change (New language, changed existing Etc.)**
- Ran `EncodingNormalizer.exe` on all files
- Added new Badges (https://github.com/northwood-studios/SCPSL-Translations/pull/1294)
- Updated 939 and 079 HUD (https://github.com/northwood-studios/SCPSL-Translations/pull/1288)
- Updated LoadingHints (https://github.com/northwood-studios/SCPSL-Translations/pull/1303)
- Updated candy descriptions
- Updated LCZ decontamination subtitles (excluding the first and the last one)

# **Checklist:**
- [ ] I have tested the translation changes ingame
- [X] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [X] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  *( Exception to this is when edits has __only__ been made in-browser. If this is the case, also* ***check this box*** *)*
- [X] I have made sure that I'm **not** submitting a duplicate translation 
- [X] I have made sure my addition follows guidelines described in the [CONTRIBUTING.md](https://github.com/northwood-studios/SCPSL-Translations/blob/master/.github/CONTRIBUTING.md) file
- [X] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation
> For more information regarding game translations, please visit our [README](https://github.com/northwood-studios/SCPSL-Translations/blob/master/README.md) page.

### CODE REVIEW

File: Translations/pl/LoadingHints.txt [hunk 1]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
 Granat może być użyty do wyważenia drzwi.
 Granat może być lekko podrzucony poprzez użycie prawego przycisku myszy.
 Micro H.I.D jest w stanie prędko zneutralizować większość podmiotów SCP, lecz ma ograniczony zasób mocy. Użyj go mądrze!
-Micro H.I.D potrzebuje 5 sekund aby się naładować do wystrzału.
+Micro H.I.D potrzebuje pięciu sekund aby w pełni przygotować się do wystrzału.
```
Comment (zabszk) [inline Translations/pl/LoadingHints.txt:30]:
```suggestion
Micro H.I.D potrzebuje pięciu sekund, aby w pełni przygotować się do wystrzału.
```

File: Translations/pl/LoadingHints.txt [hunk 2]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
 Tylko cztery butelki SCP-207 mogą zostać wypite, zauważalnie zwiększając twą prędkość ruchu.
 SCP-500 nie tylko odnawia całe zdrowie, ale również usuwa większość negatywnych efektów oraz daje dodatkową regenerację.
-Nosiciel SCP-268 jest niewidzialny dla każdego, prócz SCP-079 i SCP-939.
+Osoba, która jest pod wpływem SCP-268 jest niewidzialna dla każdego, lecz SCP-079 oraz SCP-939 dalej mogą ją zauważyć.
 SCP-268 nie uratuje cię przed SCP-096 gdy spojrzysz się na jego twarz.
 SCP-018 jest ogromnie nieprzedywidalna. Gdy rzucona, warto znaleźć jakieś schronienie.
 SCP-018 może być użyta do zatrzymania ścigającego gdy rzucona w korytarzu.
+SCP-330 może dać sześć unikalnych cukierków, z czego każdy ma swój własny efekt. Nie bierz więcej niż dwa!
+SCP-1853 pozwala na szybsze przeładowanie broni i zmniejsza odrzut broni podczas strzelania.
+SCP-2176 może zostać użyty do zablokowania oraz odblokowywania drzwi w danym pomieszczeniu, również odblokowywuje drzwi zablokowane przez SCP-079.
```
Comment (zabszk) [inline Translations/pl/LoadingHints.txt:43]:
```suggestion
SCP-2176 może zostać użyty do zablokowania oraz odblokowywania drzwi w danym pomieszczeniu. Odblokowywuje również drzwi zablokowane przez SCP-079.
```

File: Translations/pl/LoadingHints.txt [hunk 3]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+SCP-2176 może zostać użyty do oszołomienia SCP-079, jeśli zostanie użyty w pomieszczeniu, w którym on się znajduje.
+SCP-244 spowalnia prędkość z jaką można podnosić przedmioty i powstrzymuje SCP przed regeneracją swoich tarcz.
```
Comment (zabszk) [inline Translations/pl/LoadingHints.txt:45]:
```suggestion
SCP-244 zmniejsza prędkość, z jaką można podnosić przedmioty i powstrzymuje SCP przed regeneracją swoich tarcz.
```

File: Translations/pl/LoadingHints.txt [hunk 4]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+SCP-1576 pozwala żywym graczom na kontakt z obserwatorami przez krótki czas.
 SCP-914 może zmienić każdy przedmiot, doliczając w tym broń oraz amunicję.
 SCP-914 może naładować radia oraz Micro H.I.D.
+Za pomocą SCP-914 możesz wytworzyć wiele unikalnych przedmiotów, których nie można zdobyć nigdzie indziej. Zachęcamy do eksperymentowania!
```
Comment (zabszk) [inline Translations/pl/LoadingHints.txt:49]:
```suggestion
Za pomocą SCP-914 możesz wytworzyć wiele unikatowych przedmiotów, których nie można zdobyć nigdzie indziej. Zachęcamy do eksperymentowania!
```

File: Translations/pl/LoadingHints.txt [hunk 5]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
 SCP-079 rozpoczyna będąc słabym, lecz z czasem staje się bardziej potężniejszy.
-SCP-079 może zablokowywać drzwi, gasić światła oraz aktywowywać bramy tesli.
+SCP-079 może zablokować drzwi, gasić światła oraz aktywować wyładowania elektryczne na bramach tesli.
 Brama tesli staje się aktywna gdy SCP-079 zaczyna ją obserwować.
-SCP-079 nie może gasić świateł w pomieszczeniu z wyważonymi drzwiami.
 SCP-079 może zostać zabezpieczony poprzez znalezienie generatorów w Strefie Wyższego Nadzoru oraz przycisku w jego celi przechowawczej.
-Kamera używana przez SCP-079 zaświeci się na niebiesko.
+SCP-079 może zmieniać przybliżenie swoich kamer za pomocą kółka myszy.
+Im dłużej SCP-079 blokuje drzwi, tym więcej mocy będzie potrzebował, aby utrzymać je zablokowane.
+SCP-079 odblokowuje skaner wyłomów od czwartego poziomu. Jeżeli wykryje on w pomieszczeniu ukrywającego się człowieka, odpali się alarm.
```
Comment (zabszk) [inline Translations/pl/LoadingHints.txt:56]:
```suggestion
SCP-079 odblokowuje skaner wyłomów od czwartego poziomu. Jeżeli wykryje on w pomieszczeniu ukrywającego się człowieka, aktywuje się alarm.
```

File: Translations/pl/LoadingHints.txt [hunk 6]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+Kamera używana przez SCP-079 zaświeci się na niebiesko. Miej się na baczności! Nigdy nie wiadomo, kiedy będzie obserwować.
 SCP-173 bez problemu może się poruszać w ciemnych pomieszczeniach, nawet gdy jest obserwowany. Warto przynieść latarkę!
 SCP-173 może w każdej chwili zabić od tyłu, gdy nie jest obserwowany.
 SCP-173 może stworzyć kałużę spowalniającą ludzi przechodzących przez nią, stając się bronią obszarową.
-SCP-106 jest odporny na pociski, lecz nie na wybuchy lub prąd.
-SCP-106 może zostać zabezpieczony poprzez użycie łamacza kości udowej, znajdującej się w jego celi przechowawczej.
 SCP-106 może przechodzić przez drzwi oraz szyby.
-SCP-096 jest najbardziej wystawiony na ataki gdy zacznie się uspokajać.
-SCP-096 może zaszarżować podczas szału, wyważając drzwi oraz zabijając cele.
+SCP-106 jest w stanie dostrzec osoby na niewielki zasięg, gdy używa swojej umiejętności prześladowania.
```
Comment (zabszk) [inline Translations/pl/LoadingHints.txt:63]:
```suggestion
SCP-106 jest w stanie dostrzec osoby w pobliżu, gdy używa swojej umiejętności prześladowania.
```

File: Translations/pl/LoadingHints.txt [hunk 7]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+SCP-106 może otworzyć mapę pobliskich pomieszczeń, do których może się teleportować (domyślnie pod "TAB")
```
Comment (zabszk) [inline Translations/pl/LoadingHints.txt:64]:
```suggestion
SCP-106 może otworzyć mapę pobliskich pomieszczeń, do których może się teleportować (domyślnie pod "TAB").
```

File: Translations/pl/SCP079_HUD.txt [hunk 8]
```diff
@@ -72,3 +72,23 @@ Praktycznie niezniszczalny:Trzy generatory znajdują się w Strefie Wyższego Na
 Nielimitowany dostęp:Możesz kontrolować wiele systemów w placówce, pomagając innym podmiotom SCP. W prawym górnym rogu twojego ekranu wyświetlane są dostępne umiejętności. Każda z nich zużywa energię, która z czasem się regeneruje.
 Głębokie uczenie:Pomaganie podmiotom SCP jest wynagradzane w postaci doświadczenia, potrzebnego do uzyskania następnego poziomu dostępu. Każdy kolejny poziom zapewnia więcej energii oraz nowe umiejętności.
 Obserwacja:Będąc wszędzie i zarazem nigdzie posiadasz unikatową perspektywę całej placówki. Możesz zaznaczać lokacje, ludzi oraz obiekty zainteresowań, pomagając członkom drużyny w rzeźni.
+$SKANER WYŁOMÓW$ jest dostępny — wybierz drużyny/strefy na $mapie Placówki$ aby cyklicznie skanować w poszukiwaniu pozostałych ocalałych.
```
Comment (zabszk) [inline Translations/pl/SCP079_HUD.txt:75]:
```suggestion
$SKANER WYŁOMÓW$ jest dostępny — wybierz drużyny/strefy na $mapie Placówki$, aby cyklicznie skanować w poszukiwaniu pozostałych ocalałych.
```

File: Translations/pl/SCP079_HUD.txt [hunk 9]
```diff
@@ -72,3 +72,23 @@ Praktycznie niezniszczalny:Trzy generatory znajdują się w Strefie Wyższego Na
...
+ZAWIESZONO REGENERACJE ENERGII
```
Comment (zabszk) [inline Translations/pl/SCP079_HUD.txt:76]:
```suggestion
ZAWIESZONO REGENERACJĘ ENERGII
```

File: Translations/pl/SCP079_HUD.txt [hunk 10]
```diff
@@ -72,3 +72,23 @@ Praktycznie niezniszczalny:Trzy generatory znajdują się w Strefie Wyższego Na
...
+ZMNIEJSZONO REGENERACJE ENERGII ([precentage])
```
Comment (zabszk) [inline Translations/pl/SCP079_HUD.txt:77]:
```suggestion
ZMNIEJSZONO REGENERACJĘ ENERGII ([precentage])
```

File: Translations/pl/SCP079_HUD.txt [hunk 11]
```diff
@@ -72,3 +72,23 @@ Praktycznie niezniszczalny:Trzy generatory znajdują się w Strefie Wyższego Na
...
+WYBRANE STREFY: [cnt]
+WYBIERZ STREFY, ABY JE ZESKANOWAĆ
+Wyniki skanowania: $Nie wykryto nieruchomych celów$.
+Ponawianie próby za [time] sekund...
+Wyniki skanowania: $[role]$ wykryto w ‘$[camera_name]$’ ([zone]).
+NASTĘPNY SKAN: [time] s
+Skanowanie...
+SKANOWANIE ZAWIESZONE
+WYBIERZ DRUŻYNY DO WYKRYCIA
+Skaner wyłomów wykrywa pozostałe cele, które przebywają przez dłuższy czas w tym samym obszarze.
```
Comment (zabszk) [inline Translations/pl/SCP079_HUD.txt:93]:
```suggestion
Skaner wyłomów wykrywa pozostałe cele, które przebywają przez dłuższy czas na tym samym obszarze.
```

File: Translations/pl/SCP939_HUD.txt [hunk 12]
```diff
@@ -27,13 +27,20 @@ Ustawiono źródło dźwięku!
...
 Udawanie
 Strzały
-Inne
-Poprzednie
-Następne
+Różne
+-
+-
+Role
+Skrót  [key]
```
Comment (zabszk) [inline Translations/pl/SCP939_HUD.txt:41]:
```suggestion
Skrót [key]
```

**Review by zabszk — CHANGES_REQUESTED** [review]

File: Translations/pl/LoadingHints.txt [hunk 13]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
 Granat może być użyty do wyważenia drzwi.
 Granat może być lekko podrzucony poprzez użycie prawego przycisku myszy.
 Micro H.I.D jest w stanie prędko zneutralizować większość podmiotów SCP, lecz ma ograniczony zasób mocy. Użyj go mądrze!
-Micro H.I.D potrzebuje 5 sekund aby się naładować do wystrzału.
+Micro H.I.D potrzebuje pięciu sekund, aby w pełni przygotować się do wystrzału.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:30]:
```suggestion
Micro H.I.D potrzebuje pięciu sekund, aby przygotować się do wystrzału.
```

File: Translations/pl/LoadingHints.txt [hunk 14]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
 Trzymanie prawego przycisku myszy przytrzyma Micro H.I.D w stanie gotowości, zużywając powoli jego zasób mocy.
 Micro H.I.D jest bardzo głośny podczas ładowania. Miej pewność, że twój cel nie zdąży uciec.
 Dźwięk ładowania Micro H.I.D jest w stanie wystraszyć i popsuć plan przeciwnikom.
 Butelka SCP-207 nie tylko zwiększa prędkość ruchu, ale również odnawia małą ilość zdrowia.
 Tylko cztery butelki SCP-207 mogą zostać wypite, zauważalnie zwiększając twą prędkość ruchu.
 SCP-500 nie tylko odnawia całe zdrowie, ale również usuwa większość negatywnych efektów oraz daje dodatkową regenerację.
-Nosiciel SCP-268 jest niewidzialny dla każdego, prócz SCP-079 i SCP-939.
+Osoba, która jest pod wpływem SCP-268 jest niewidzialna dla każdego, lecz SCP-079 oraz SCP-939 dalej mogą ją zauważyć.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:37]:
```suggestion
Osoba pod wpływem SCP-268 jest niewidzialna dla każdego, lecz SCP-079 oraz SCP-939 dalej mogą ją zauważyć.
```

File: Translations/pl/LoadingHints.txt [hunk 15]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
 SCP-268 nie uratuje cię przed SCP-096 gdy spojrzysz się na jego twarz.
 SCP-018 jest ogromnie nieprzedywidalna. Gdy rzucona, warto znaleźć jakieś schronienie.
 SCP-018 może być użyta do zatrzymania ścigającego gdy rzucona w korytarzu.
+SCP-330 może dać sześć unikalnych cukierków, z czego każdy ma swój własny efekt. Nie bierz więcej niż dwa!
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:41]:
```suggestion
SCP-330 zawiera sześć rodzajów cukierków, z czego każdy ma swój własny efekt. Nie bierz więcej niż dwa!
```

File: Translations/pl/LoadingHints.txt [hunk 16]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+SCP-1853 pozwala na szybsze przeładowanie broni i zmniejsza odrzut broni podczas strzelania.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:42]:
```suggestion
SCP-1853 pozwala na szybsze przeładowanie broni i zmniejsza odrzut podczas strzelania.
```

File: Translations/pl/LoadingHints.txt [hunk 17]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+SCP-2176 może zostać użyty do zablokowania oraz odblokowywania drzwi w danym pomieszczeniu. Odblokowywuje również drzwi zablokowane przez SCP-079.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:43]:
```suggestion
SCP-2176 może zostać użyty do blokowania oraz odblokowywania drzwi w danym pomieszczeniu. Działa również na drzwi pod kontrolą SCP-079.
```

File: Translations/pl/LoadingHints.txt [hunk 18]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+SCP-2176 może zostać użyty do oszołomienia SCP-079, jeśli zostanie użyty w pomieszczeniu, w którym on się znajduje.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:44]:
```suggestion
SCP-2176 może zostać użyty do oszołomienia SCP-079, jeśli zostanie użyty w pomieszczeniu które on obserwuje.
```

File: Translations/pl/LoadingHints.txt [hunk 19]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+SCP-244 zmniejsza prędkość, z jaką można podnosić przedmioty i powstrzymuje SCP przed regeneracją swoich tarcz.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:45]:
```suggestion
SCP-244 zmniejsza prędkość, z jaką można podnosić przedmioty i powstrzymuje regenerację tarcz innych SCP.
```

File: Translations/pl/LoadingHints.txt [hunk 20]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+SCP-1576 pozwala żywym graczom na kontakt z obserwatorami przez krótki czas.
 SCP-914 może zmienić każdy przedmiot, doliczając w tym broń oraz amunicję.
 SCP-914 może naładować radia oraz Micro H.I.D.
+Za pomocą SCP-914 możesz wytworzyć wiele unikatowych przedmiotów, których nie można zdobyć nigdzie indziej. Zachęcamy do eksperymentowania!
 SCP-079 rozpoczyna będąc słabym, lecz z czasem staje się bardziej potężniejszy.
-SCP-079 może zablokowywać drzwi, gasić światła oraz aktywowywać bramy tesli.
+SCP-079 może zablokować drzwi, gasić światła oraz aktywować wyładowania elektryczne na bramach tesli.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:51]:
```suggestion
SCP-079 może blokować drzwi, gasić światła oraz aktywować wyładowania elektryczne na bramach tesli.
```

File: Translations/pl/LoadingHints.txt [hunk 21]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
 Brama tesli staje się aktywna gdy SCP-079 zaczyna ją obserwować.
-SCP-079 nie może gasić świateł w pomieszczeniu z wyważonymi drzwiami.
 SCP-079 może zostać zabezpieczony poprzez znalezienie generatorów w Strefie Wyższego Nadzoru oraz przycisku w jego celi przechowawczej.
-Kamera używana przez SCP-079 zaświeci się na niebiesko.
+SCP-079 może zmieniać przybliżenie swoich kamer za pomocą kółka myszy.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:54]:
```suggestion
SCP-079 może zmieniać przybliżenie kamer za pomocą kółka myszy.
```

File: Translations/pl/LoadingHints.txt [hunk 22]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+Im dłużej SCP-079 blokuje drzwi, tym więcej mocy będzie potrzebował, aby utrzymać je zablokowane.
+SCP-079 odblokowuje skaner wyłomów od czwartego poziomu. Jeżeli wykryje on w pomieszczeniu ukrywającego się człowieka, aktywuje się alarm.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:56]:
```suggestion
SCP-079 odblokowuje wykrywacz wyłomów od czwartego poziomu. Jeżeli wykryje on w pomieszczeniu ukrywającego się człowieka, aktywuje się alarm.
```

File: Translations/pl/LoadingHints.txt [hunk 23]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+SCP-079 może aktywować Głowicę Alfa od piątego poziomu. Jest on w stanie przetrwać wybuch głowicy, jeśli to on sam aktywował procedurę detonacji placówki.
+Kamera używana przez SCP-079 zaświeci się na niebiesko. Miej się na baczności! Nigdy nie wiadomo, kiedy będzie obserwować.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:58]:
```suggestion
Kamera używana przez SCP-079 świeci się na niebiesko. Miej się na baczności - nigdy nie wiadomo kto cię obserwuje...
```

File: Translations/pl/LoadingHints.txt [hunk 24]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
 SCP-173 może stworzyć kałużę spowalniającą ludzi przechodzących przez nią, stając się bronią obszarową.
-SCP-106 jest odporny na pociski, lecz nie na wybuchy lub prąd.
-SCP-106 może zostać zabezpieczony poprzez użycie łamacza kości udowej, znajdującej się w jego celi przechowawczej.
 SCP-106 może przechodzić przez drzwi oraz szyby.
-SCP-096 jest najbardziej wystawiony na ataki gdy zacznie się uspokajać.
-SCP-096 może zaszarżować podczas szału, wyważając drzwi oraz zabijając cele.
+SCP-106 jest w stanie dostrzec osoby w pobliżu, gdy używa swojej umiejętności prześladowania.
+SCP-106 może otworzyć mapę pobliskich pomieszczeń, do których może się teleportować (domyślnie pod "TAB").
+Umiejętności SCP-106 zależą od zasobu znanego jako "wigor", który wyczerpuje się z każdym użyciem umiejętności.
+SCP-106 jest najbardziej wrażliwy na ataki podczas zaraz po wynurzeniu się z ziemi.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:66]:
```suggestion
SCP-106 jest najbardziej wrażliwy na ataki podczas wynurzenia się z ziemi.
```

File: Translations/pl/LoadingHints.txt [hunk 25]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+SCP-096 jest najbardziej wrażliwy na ataki, gdy zaczyna się uspokajać.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:67]:
```suggestion
SCP-096 jest najbardziej wrażliwy na ataki, gdy uspokaja się.
```

File: Translations/pl/LoadingHints.txt [hunk 26]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+SCP-096 potrafi szarżować, gdy jest wściekły - wyłamuje drzwi i zabija cele.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:68]:
```suggestion
SCP-096 gdy wściekły, potrafi szarżować - wyłamuje wtedy drzwi i zabija cele.
```

File: Translations/pl/LoadingHints.txt [hunk 27]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
 SCP-096 podczas szału będzie wyważać drzwi oraz przeciskać się przez bramy.
-Pancerz nie chroni przed ugryzieniami SCP-939.
-SCP-939 nakłada amnezję na swych ofiarach, pogarszając im pole widzenia.
-SCP-939 może nieskończenie biegać oraz jest całkowicie cichy podczas powolnego chodzenia.
-SCP-939 jest cichy podczas chodzenia, dzięki czemu w pełni wpasowuje się w rolę drapieżnika.
+SCP-939 potrafi naśladować głosy graczy, których wcześniej zabił.
+SCP-939 może wypuścić amnezyjny obłok. Ludzie znajdujący się pod jego wpływem nie zauważą SCP-939, dopóki on sam nie zaatakuje.
+Użycie adrenaliny neutralizuje działanie obłoku amnezyjnego SCP-939.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:72]:
```suggestion
Użycie adrenaliny neutralizuje działanie amnezyjnego obłoku SCP-939.
```

File: Translations/pl/LoadingHints.txt [hunk 28]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+Granaty błyskowe są szczególnie skuteczne przeciwko SCP-939.
+Im niższa jest wytrzymałość SCP-939, tym gorzej widzi on graczy.
+Im dłużej SCP-939 jest skupiony, tym skuteczniejsza staje się jego Ultraczułość.
 Kroki podmiotów SCP zauważalnie się różnią od ludzkich. Słuchaj uważnie!
-SCP-049 może natychmiastowo zabić każdego człowieka, który znajdzie się w jego zasięgu.
+Bycie zaatakowanym przez SCP-049 powoduje zatrzymania akcji serca. Zdrowie i wytrzymałość szybko się wyczerpują podczas trwania ataku.
+SCP-049 może natychmiast zabić każdego człowieka będącego już pod wpływem efektu akcji zatrzymania serca.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:78]:
```suggestion
SCP-049 może natychmiast zabić każdego człowieka będącego już pod wpływem efektu zatrzymania akcji serca.
```

File: Translations/pl/LoadingHints.txt [hunk 29]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
+Zatrzymanie akcji serca gwałtownie pozbawia twoje zdrowie. Będziesz potrzebował wielu środków medycznych, jeżeli chcesz go przetrwać.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:79]:
```suggestion
Zatrzymanie akcji serca gwałtownie pozbawia cię zdrowia. Będziesz potrzebował wiele środków medycznych, jeżeli chcesz je przetrwać.
```

File: Translations/pl/LoadingHints.txt [hunk 30]
```diff
@@ -24,40 +27,58 @@ Bronie Rebelii Chaosu są potężne, lecz amunicja do nich nie znajduje się w p
...
 SCP-049 może wskrzeszać tylko świeże zwłoki.
+SCP-049 może dać drobną tarczę dla podmiotów SCP-049-2, które znajdują się w pobliżu.
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:81]:
```suggestion
SCP-049 może dać słabą tarczę dla podmiotów SCP-049-2, które znajdują się w pobliżu.
```

File: Translations/pl/SCP079_HUD.txt [hunk 31]
```diff
@@ -72,3 +72,23 @@ Praktycznie niezniszczalny:Trzy generatory znajdują się w Strefie Wyższego Na
 Nielimitowany dostęp:Możesz kontrolować wiele systemów w placówce, pomagając innym podmiotom SCP. W prawym górnym rogu twojego ekranu wyświetlane są dostępne umiejętności. Każda z nich zużywa energię, która z czasem się regeneruje.
 Głębokie uczenie:Pomaganie podmiotom SCP jest wynagradzane w postaci doświadczenia, potrzebnego do uzyskania następnego poziomu dostępu. Każdy kolejny poziom zapewnia więcej energii oraz nowe umiejętności.
 Obserwacja:Będąc wszędzie i zarazem nigdzie posiadasz unikatową perspektywę całej placówki. Możesz zaznaczać lokacje, ludzi oraz obiekty zainteresowań, pomagając członkom drużyny w rzeźni.
+$SKANER WYŁOMÓW$ jest dostępny — wybierz drużyny/strefy na $mapie Placówki$, aby cyklicznie skanować w poszukiwaniu pozostałych ocalałych.
+ZAWIESZONO REGENERACJĘ ENERGII
+ZMNIEJSZONO REGENERACJĘ ENERGII ([precentage])
```
[the remark this answers is not in the document]
Comment (zabszk) [reply Translations/pl/SCP079_HUD.txt]: I think it should be without `(` and `)` in this case.

File: Translations/pl/SCP939_HUD.txt [hunk 32]
```diff
@@ -27,13 +27,20 @@ Ustawiono źródło dźwięku!
...
-Następne
+Różne
+-
+-
+Role
+Skrót [key]
+Odtwórz wybrany dźwięk (każdy go usłyszy)
+Zatrzymaj odtwarzanie
+Przeciągnij, aby przyciąć nagranie
+Zaznacz ulubione. Nagrania oznaczone jako ulubione przypisywane są do skrótów.
```
Comment (MichalPetryka) [inline Translations/pl/SCP939_HUD.txt:45]:
```suggestion
Zaznacz ulubione. Nagrania oznaczone jako ulubione zostaną przypisane do skrótów.
```

File: Translations/pl/LoadingHints.txt [hunk 33]
```diff
@@ -1,11 +1,14 @@
 Możesz rozbroić gracza poprzez interakcję z nim podczas trzymania broni palnej.
 Placówka posiada cztery strefy: Strefę Niższego Nadzoru, Strefę Wyższego Nadzoru, Strefę Wejściową oraz powierzchnię.
 Aby uciec, naukowcy i klasa D muszą się dostać na powierzchnię.
+Ochroniarze są kiepscy w pojedynkę, ale są mocniejsi w grupie. W grupie siła!
```
Comment (MichalPetryka) [inline Translations/pl/LoadingHints.txt:4]:
```suggestion
Ochroniarze są kiepscy w pojedynkę, jednak razem są mocniejsi. W grupie siła!
```

File: Translations/pl/SCP939_HUD.txt [hunk 34]
```diff
@@ -1,8 +1,8 @@
 Naciśnij [key_attack] lub [key_jump], aby wyskoczyć.
 Odsłuchaj głos (tylko ty to usłyszysz)
-PODGLĄD
+-
 Usuń głos (przytrzymaj)
-Transmituj głos (każdy to usłyszy)
+Odtwórz głos (każdy go usłyszy)
```
Comment (MichalPetryka) [inline Translations/pl/SCP939_HUD.txt:5]:
```suggestion
Odtwórz głos (usłyszy go każdy)
```

File: Translations/pl/SCP330.txt [hunk 35]
```diff
@@ -3,20 +3,20 @@ Tęczowy cukierek
...
+Przytłaczający zapach cytryny sprawia, że się krzywisz.
+Pozwala na szybsze poruszanie się i dłuższy bieg przez krótki okres czasu.
 Fioletowy cukierek
-Posiada soczysty zapach winogron. Twoje usta zaczynają ślinieć.
+Posiada soczysty zapach winogron. Patrząc na niego zaczynasz się ślinić.
 Przyznaje tymczasową odporność na obrażenia oraz powolną regenerację zdrowia.
 Czerwony cukierek
-Silny zapach wiśni wypełnia pomieszczenie. Jest troszkę <i>za bardzo</i> silny.
-Szybko przywraca zdrowie.
+Silny zapach wiśni wypełnia pomieszczenie. Jest troszkę <i>za mocny</i>.
```
Comment (MichalPetryka) [inline Translations/pl/SCP330.txt:12]:
```suggestion
Silny zapach wiśni wypełnia pomieszczenie. Jest lekko <i>przytłaczający</i>.
```

**Review by MichalPetryka — APPROVED** [review]

**Review by zabszk — APPROVED** [review]

**Pull request merged**